### PR TITLE
make bot more human

### PR DESCRIPTION
**Pull request by lifangjie**
add random 0-500 milliseconds delay

### CODE REVIEW

**Comment by icbrg** [comment]
All random data possible it's a good idea

**Comment by panthernet** [comment]
Small trick to make random a better random :)
new Random(Guid.NewGuid().GetHashCode())

**Comment by AdamWeishaupt1776** [comment]
I still think all of these random delays should be an option

e.g. UseRandomDelays: true/false

**Comment by panthernet** [comment]
And random deviation should be configurable too

**Comment by lifangjie** [comment]
I updated this pr, random deviation is configurable now.

File: PoGo.NecroBot.Logic/Utils/Statistics.cs [hunk 1]
```diff
@@ -35,6 +37,29 @@ public void Dirty(Inventory inventory)
             DirtyEvent?.Invoke();
         }
 
+        public static Task Tesk { get; private set; }
```
Comment (AdamWeishaupt1776) [inline PoGo.NecroBot.Logic/Utils/Statistics.cs]: is "Tesk" a typo?

Comment (lifangjie) [inline PoGo.NecroBot.Logic/Utils/Statistics.cs]: sorry, it's a terrible mistake, fixed already

**Comment by NecronomiconCoding** [comment]
now you guys can update for the new changes

**Pull request closed without merging**